src/components/vuestic-components/vuestic-breadcrumbs/VuesticBreadcrumbs.vue

```diff
@@ -0,0 +1,65 @@
+<template>
+  <vuestic-widget class="no-padding no-v-padding">
+    <div class='vuestic-breadcrumbs'>
+      <router-link :to="'/'" class="crumb">{{'breadcrumbs.home' | translate}}</router-link>
+      <router-link
+              v-for="(item, index) in breadcrumbs"
+              :to="{path: item.path}"
+              :key="index"
+              class="crumb"
+              v-bind:class="{ disabled: !item.path }">
+        {{item.meta.title | translate}}
+      </router-link>
+    </div>
+  </vuestic-widget>
+
+</template>
+
+<script>
+  export default {
+    name: 'vuestic-breadcrumbs',
+    computed: {
+      breadcrumbs () {
+        return this.$store.getters.breadcrumbs(this.$route.name)
```

Comment:
It's better not to use store inside vuesptic-components. The reason - components must be the building bricks of the app, no matter of app's structure. It's better to implement the method right here and delete it from store completely.

Reply:
Also, routes list and current route might be passed as properties, so breadcrumbs would be not dependent of vue-router

Reply:
So, what approach do you prefer?

Reply:
Props to breadcrumbs, no store usage, method inside of the component

Reply:
@smaliankou the thing is, if we figure out how to get rid of bootstrap styles dependency - our cue-components folder might become a separate lib :)

Reply:
What method inside of the component do you mean? We just can pass breadcrumbs array to the component

Reply:
Props to breadcrumbs are more pretty. I agree
I don't know why I used the store inside the component :)

Reply:
@smaliankou findInNestedByName

Reply:
@smaliankou no probs, anyway - everything is nice  :)

Reply:
I think we can use this function out of the component and pass only specific router items. A highly simplistic example
["Home", "Forms", "Form elements"]
Although I need to think